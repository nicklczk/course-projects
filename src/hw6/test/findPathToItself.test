# Tests search algorithm from a node to itself

# Create graph
LoadGraph friends champions.tsv

# find path to itself
FindPath friends Vincent Vincent