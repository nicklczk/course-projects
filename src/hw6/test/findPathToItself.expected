# Tests search algorithm from a node to itself

# Create graph
loaded graph friends

# find path to itself
path from Vincent to Vincent: